src/tools/perf/lib/libperf.c

```diff
@@ -1414,7 +1393,9 @@ static void ucp_perf_test_destroy_workers(ucx_perf_context_t *perf)
     unsigned i;
 
     for (i = 0; i < perf->params.thread_count; i++) {
-        ucp_worker_destroy(perf->ucp.tctx[i].thread_worker);
+        if (perf->ucp.tctx[i].perf.ucp.worker != NULL) {
+            ucp_worker_destroy(perf->ucp.tctx[i].perf.ucp.worker);
+        }
```

Comment:
can we use ucp_perf_test_destroy_workers()?

Reply:
what do you mean? this is ucp_perf_test_destroy_workers()...

Reply:
right:)